cdc/owner/ddl_sink.go

```diff
@@ -54,6 +54,7 @@ type DDLSink interface {
 	// the caller of this function can call again and again until a true returned
 	emitDDLEvent(ctx context.Context, ddl *model.DDLEvent) (bool, error)
 	emitSyncPoint(ctx context.Context, checkpointTs uint64) error
+	emitBootstarp(ctx context.Context, bootstrap *model.DDLEvent) error
```

Comment:
It seems a typo issue: bootstarp --> bootstrap?
BTW, Please add a comment for this function

Reply:
It looks that `emitBootstrap` is almost the same as `emitDDLEvent`.

IMPO, they should share the same interface method.

Reply:
emitBootstrap is a blocking interface, whereas emitDDLEvent is an asynchronous interface.